Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -1287,7 +1287,9 @@ void WebChromeClient::setUpPlaybackControlsManager(HTMLMediaElement& mediaElemen
 
 void WebChromeClient::clearPlaybackControlsManager()
 {
-    protectedPage()->playbackSessionManager().clearPlaybackControlsManager();
+    // FIXME: call protectedPage() once it returns a RefPtr.
```

Comment:
This FIXME comment is not accurate. We still wouldn't call `protectedPage()`, `protectedFoo()` functions are only used to avoid local variables. However, here we have a local variable we use for the null check. So we'd either use `if (RefPtr page = m_page.get())` like you did or `if (RefPtr page = this->page())`